src/engraving/dom/cmd.cpp

```diff
@@ -2522,6 +2522,7 @@ void Score::resetDefaults()
             }
         }
     }
+    score()->doLayout();
```

Comment:
`score()` is redundant here, because `this` _is_ already the score :)

But actually I think this comment from Jojo makes sense: https://github.com/musescore/MuseScore/issues/19625#issuecomment-1748619616
Why are we even reading the measure stretch for old scores, if the first thing we'll do is discarding it?

Reply:
Ok, I'll see if not reading the stem length, trem offset, etc.  as well is a viable option so we can get rid of the `Score::resetDefaults()` method completely.